lib/logstash/devutils/rspec/spec_helper.rb

```diff
@@ -43,6 +43,7 @@ def []=(str, value)
 end
 
 RSpec.configure do |config|
+  config.include LogStashHelper
```

Comment:
Hm, I think it's more explicit to do both here as it clearly state the intention of both extending and including.